src/shared/utils.ts

```diff
@@ -43,6 +43,14 @@ export function isWebProject(projectPath: string): boolean {
     return projectFileText.toLowerCase().indexOf('sdk="microsoft.net.sdk.web"') >= 0;
 }
 
+export function isWebAssemblyProject(projectPath: string): boolean {
+    const projectFileText = fs.readFileSync(projectPath, 'utf8');
```

Comment:
>const projectFileText = fs.readFileSync(projectPath, 'utf8'); [](http://example.com/codeflow?start=4&length=61)

Instead of reading the file a second time, I would suggest changing `isWebProject` to `getProjectFlags` and returning both.

Reply:
done